github/VisualStudio
# Ignore whitespace in Open from GitHub search box

We should ignore whitespace when searching for a repository.

### What this PR does

- Ignore any whie-space that appears before or after the filter/URL
- Check that we ignore white-space before and after filter

### How to test

1. Launch VS and click 'Clone or check out code' in Start Window
2. Click 'GitHub' under 'Browse a repository' and sign in
3. Input the repo name/location without any blank in searching box
4. Add a blank space before or after the filter
5. Check that the filtered repository(s) still appear

Fixes #2350
